src/main/kotlin/no/nav/familie/klage/behandling/BehandlingService.kt

```diff
@@ -100,4 +104,28 @@ class BehandlingService(
             "Det eksisterer allerede en klagebehandling som ikke er ferdigstilt på fagsak med id=${fagsak.id}"
         }
     }
+
+    fun henleggBehandling(behandlingId: UUID, henlagt: HenlagtDto) {
+        val behandling = hentBehandling(behandlingId)
+
+        validerKanHenleggeBehandling(behandling)
+
+        val henlagtBehandling = behandling.copy(
+            henlagtÅrsak = henlagt.årsak,
+            resultat = BehandlingResultat.HENLAGT,
+            steg = BEHANDLING_FERDIGSTILT,
+            status = FERDIGSTILT
+        )
+
+        // TODO: Legg til historikkinnslag
+        // TODO: Ferdigstill oppgave
```

Comment:
Denne todoen er vel ikke nødvendig?

Reply:
Fordi det er en generell todo på all ferdigstilling? Den er relevant fordi det "skal" finnes en oppgave forbundet med behandlingen - og den bør jo ferdigstilles?

Reply:
Vi trenger nog en ferdigstill oppgave for de som blir ferdigstilte på vanlig måte og :-)

Reply:
Opprettet en generell oppgave for dette, https://favro.com/organization/98c34fb974ce445eac854de0/a64c6aad9b0d61ef6c0290bd?card=Tea-9702

* Opprette oppgave når vi lager en klagebehandling (både via journalføring og manuellt opprettet)
* Lukke oppgave ved henleggelse
* Lukke oppgave ved normal ferdigstilling